content/en/apps/reference/app-settings/transitions.md

```diff
@@ -671,4 +672,63 @@ If this configuration is not set then the message defaults to what is set in the
     ]
   }
 ]
-```
\ No newline at end of file
+```
+
+### create_user_for_contacts
+
+An existing offline user can be replaced on a device so that a new user can use that device without needing to immediately sync with the server. _Available since 3.17.x._
+
+#### Example scenario
+
+Imagine a CHW is leaving the program, and the CHW's device is returned to their supervisor. The supervisor wants to transfer the device to a new CHW immediately without having the device online to sync with the server.
+
+To do this, when the `create_user_for_contacts` transition is enabled, the supervisor would submit a configured user replacement form for the original user's contact on the device. This form can create a new contact for the new CHW and will trigger a client-side transition to mark the original contact as replaced. After that, the supervisor can give the device to the new CHW, and they can begin using it.
+
+Subsequent reports submitted on the device by the new CHW will be associated with the new contact. When the device is eventually able to synchronize with the server, it will be automatically logged out (as it was actually still logged in as the original user). A server-side transition will be triggered to inactivate the original user and to create a new user for the new CHW. An SMS message containing a token login link will be sent to the new CHW allowing them to login as their proper user. 
+
+#### Configuration
+
+Several configurations are required in `app_settings` to enable the `create_user_for_contacts` transition. 
+
+[Login by SMS]({{< ref "apps/reference/api#login-by-sms" >}}) must be enabled by setting the `token_login` configuration.
+
+The [`app_url` property]({{< ref "apps/reference/app-settings#app_settingsjson" >}}) must be set to the URL of the application. This is used to generate the token login link for the new user.
+
+The IDs of the app forms that should trigger the transition must be configured in the `create_user_for_contacts.replace_forms` array.
+
+##### `replace_forms`
+
+Forms that trigger the `create_user_for_contacts` transition must set the `replacement_contact_id` property to the id of the contact that should be associated with the new user. 
+
+These forms should only be submitted for the _original user's contact_. You can control the form visibility by including `user._id === contact._id` in the form properties expression. 
+
+These forms should only be accessible to offline users (replacing online users is not currently supported). This is the default in the example app form properties file ([see `replace_user.properties.json`](https://github.com/medic/cht-core/blob/master/config/default/forms/app/replace_user.properties.json)).
+
+You can prevent a user from being replaced multiple times by including `!contact.user_for_contact.replaced` in the form properties expression.
```

Comment:
Need to include a note that this will not work for situations where multiple users are associated with the same contact.